tests/test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+beautifulsoup4
```

Comment:
Did you create this by hand? Usually they have version numbers, which could be useful for making sure tests don't break in the future because of a package upgrade